modules/server/src/main/ts/bedrock/auto/Driver.ts

```diff
@@ -188,6 +203,11 @@ export const create = async (settings: DriverSettings): Promise<Driver> => {
     // Wait for the driver to start up and then start the webdriver session
     await DriverLoader.startAndWaitForAlive(driverApi, port, webdriverTimeout);
     const webdriverOptions = getOptions(port, browserName, browserFamily, settings);
+
+    console.log(
```

Comment:
This seems like the wrong logging level here. This isn't something we really want logged every time as it's just noted needed. It should only be included when doing verbose logging (or I think it's called advanced logging in bedrock).

- [ ] Change this to only log when more detailed logging is enabled or remove it entirely

Reply:
From memory, advanced logging isn't really about log level ... it's about having a logging session that allows you to use special characters to move the cursor in the output and overwrite existing lines. I can put it through advanced logging, but I didn't realise we were using it for log levels. Do we do that anywhere else? 

I would strongly discourage not logging it altogether.

It's also just one to two lines. Is that really too much? Or does it look like way too much for Firefox Headless? Examples: 

```
e.g. chrome-headless
Browser capabilities: {"browserName":"chrome","goog:chromeOptions":{"args":["--start-maximized","--disable-extensions","--dig","--goose","--blah","--headless","--remote-debugging-port=9000"]}}

e.g. chrome
Browser capabilities: {"browserName":"chrome","goog:chromeOptions":{"args":["--start-maximized","--disable-extensions","--dig","--goose","--blah"]}

e.g. firefox
Browser capabilities: {"browserName":"firefox","moz:firefoxOptions":{"args":["--dig","--goose","--blah"]}}

e.g. firefox headless
Browser capabilities: {"browserName":"firefox","moz:firefoxOptions":{"args":["--dig","--goose","--blah","-headless","-start-debugger-server=9000"],"prefs":{"devtools.debugger.remote-enabled":true,"devtools.debugger.prompt-connection":false,"devtools.chrome.enabled":true}}}
```

I'm happy to do what you want, I'm just clarifying.

Reply:
`advanced` logging is also the default, so I'm happy to change this to only log on advanced. But for clarification, someone is going to have to choose 'simple' for it not to appear.

- [ ] change to only log when mode is advanced.

Reply:
The problem is that by default we should only be providing information needed to know where to run the tests and where it's up to. You can't get anything meaningful from this logging except for debugging type information, so it makes little sense to show by default.

If `advanced` is already the default then we probably need to either add another mode or just remove this.

Reply:
Another thing worth noting is that the default capabilities aren't really meant to be known/visible, they are an internal implementation detail.

Reply:
Maybe it should only log these if there are custom capabilities since then you might want to know what the full set is? And since that is more rare it would be logged less often only in the special cases like in a docker container or will that also be too verbose?

Reply:
I think this is exactly the kind of thing that would have highlighted that Microsoft Edge Chromium doesn't have the flags, though. We can add a verbose flag if that's your preference, but not having any way of logging it without changing bedrock is going to make debugging a lot harder.

If you don't want it to be the default, then I'll need to add a verbose flag.

Reply:
- [ ] add a verbose logging flag for extra details.

Reply:
Yeah having a verbose or debugging log level would definitely be the preferred option. Worth noting if we're doing that we may just want to enable verbose logging for webdriver.io as it includes all these details and much more about the webdriver calls.